set.mm

```diff
@@ -7202,6 +7202,17 @@ _praeclarum theorema_ (splendid theorem).  (Contributed by NM,
       ( wi exp41 syl3c mpd ) AEFJABCDEFLGHIBCDEFKMNO $.
   $}
 
+  ${
```

Comment:
naming conventions:  We have some similar theorems syl1111anc, syl112anc and more in main.  The digits reflect how antecedents are grouped.  22 for example should be attached to ( ( ph /\ ps ) /\ ( ch /\ th ) ).  I wonder whether we should add a 1111 sequence to the name here.

Reply:
So you propose to change mpsyl4anc to mpsyl1111anc? This would be quite long, but would be OK for me.
Or should it be called mpsyl211anc (which would be a little bit shorter), because the first conjuncts are directly grouped?

syl1111anc seems to be the only label using the sequence 1111...

Reply:
I prefer sticking to `mpsyl4anc`. The special case of left-leaning and-trees is already specialized in simp-Nl and adNantr, and it makes sense to specialize to a linear number of lemmas rather than the exponential combinations with the sylNNNanc series.

Reply:
I am not a friend of this NNNN style either, and I don't expect many more theorems of this kind to follow (if not we simply rename them later).  That's why I used "I wonder...".  Still, the naming scheme is inconsistent, and I wanted to point that out.  I am happy if you are...

Reply:
OK, then we should leave it as it is.